ios/FluentUI/Button/Button.swift

```diff
@@ -184,22 +78,74 @@ open class Button: UIButton {
     open override var intrinsicContentSize: CGSize {
         var size = titleLabel?.systemLayoutSizeFitting(CGSize(width: proposedTitleLabelWidth == 0 ? .greatestFiniteMagnitude : proposedTitleLabelWidth, height: .greatestFiniteMagnitude)) ?? .zero
         size.width = ceil(size.width + edgeInsets.leading + edgeInsets.trailing)
-        size.height = ceil(max(size.height, style.minTitleLabelHeight) + edgeInsets.top + edgeInsets.bottom)
+        size.height = ceil(max(size.height, ButtonTokenSet.minContainerHeight(style)) + edgeInsets.top + edgeInsets.bottom)
 
         if let image = image(for: .normal) {
             size.width += image.size.width
             if #available(iOS 15.0, *) {
-                size.width += style.titleImagePadding
+                size.width += ButtonTokenSet.titleImageSpacing(style)
             }
 
             if titleLabel?.text?.count ?? 0 == 0 {
-                size.width -= style.titleImagePadding
+                size.width -= ButtonTokenSet.titleImageSpacing(style)
             }
         }
 
         return size
     }
 
+    open func initialize() {
+        layer.cornerRadius = tokenSet[.cornerRadius].float
+        layer.cornerCurve = .continuous
+
+        titleLabel?.font = UIFont.fluent(tokenSet[.titleFont].fontInfo)
+        titleLabel?.adjustsFontForContentSizeCategory = true
+
+        if #available(iOS 15, *) {
```

Comment:
Do we still target iOS 14? For how much longer?

Reply:
We have some dependencies with dependencies that haven't moved up to min iOS 15 yet, so until they're ready we're still supporting it.